src/main/java/io/fabric8/maven/docker/util/EnvUtil.java

```diff
@@ -43,7 +43,13 @@ private EnvUtil() {}
 
     // Convert docker host URL to an HTTP(s) URL
     public static String convertTcpToHttpUrl(String connect) {
-        String protocol = connect.contains(":" + DOCKER_HTTP_PORT) ? "http:" : "https:";
+        String toHTTP = System.getenv("DOCKER_TCP_TO_HTTP");
```

Comment:
Is this `DOCKER_TCP_TO_HTTP` a docker-maven-plugin configuration option or is it already defined in docker?

Reply:
If it's former, we need to update our documentation as well.

Reply:
`DOCKER_TCP_TO_HTTP` belongs to the docker-maven-plugin.

Reply:
> If it's the former, we need to update our documentation as well.

Good catch!

Reply:
@trevorgetty : Is it possible for someone from your team to pick this up? You can cherry-pick this commit and create a new PR if possible.

Reply:
@trevorgetty : That would be really appreciated! I think we can start by creating a new PR out of this and working on top of it.